edtLHD/src/test/java/fr/univtln/lhd/model/entities/dao/slots/SubjectDAOTest.java

```diff
@@ -0,0 +1,44 @@
+package fr.univtln.lhd.model.entities.dao.slots;
+
+import fr.univtln.lhd.model.entities.slots.Subject;
+import fr.univtln.lhd.model.entities.user.Admin;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.sql.SQLException;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class SubjectDAOTest {
+
+    public SubjectDAO getDAO() {
+        SubjectDAO st = null;
+        try {
+            st = new SubjectDAO();
+        } catch (SQLException e) {
+            throw new RuntimeException(e);
+        }
+        return st;
+    }
+
+
+    private Subject getSubject(){//no cons on unicity random is useless
```

Comment:
Typo  in comment unicity  -> uniqueness 

Also, I will add such a constraint in a PR soon.